.github/workflows/build_docs.yml

```diff
@@ -34,8 +34,8 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install ".[docs]"
+          curl -LsSf https://astral.sh/uv/install.sh | sh
+          uv pip install --system ".[docs]"
```

Comment:
I think it might be easier to just use `pip install`. I think `uv pip install` is for the cases in which you want to use the venv created by uv. But in ci we use just the native env anyways (which is also totally fine). Or was there another reason for you to do this?

Reply:
It's a bit faster if you use `uv pip install` (also with the download of uv with `curl`), but the difference is not that big. If you use the `--system` flag, it installs into the native env (because by default it expects a venv). For me, both ways is fine.

Reply:
I see. Then let us just keep it the way it is. I was just a bit surprised by the redundancy (as uv is also in the dev dependencies)